nsds_lab_to_nwb/nwb_builder.py

```diff
@@ -103,6 +105,17 @@ def __init__(
         logger.info('Extracting session start time...')
         self.session_start_time = self._extract_session_start_time()
 
+    def _get_source_script(self):
+        info = get_software_info()
+        if info['git_branch'] != 'main':
+            warnings.warn(f"You are currently on the {info['git_branch']} branch "
+                          f"of the {info['name']} git repository. " +
+                          "Final NWB files should be created from the main branch.")
+        source_script = (f"Created by nsds-lab-to-nwb {info['version']} "
+                         f"({info['url']}) "
+                         f"(git@{info['git_describe']})")
```

Comment:
The git hash is just a temporary solution before we start versioning. But any suggestions are welcome!